docs/cudf/source/cudf_pandas/faq.md

```diff
@@ -181,6 +181,19 @@ There are a few known limitations that you should be aware of:
    ```
 - `cudf.pandas` (and cuDF in general) is only compatible with pandas 2. Version
   24.02 of cudf was the last to support pandas 1.5.x.
+- `cudf.pandas` can interface with functions that utilize NumPy's C API, but doing so requires
+  a data transfer from device to host to ensure that the [data buffer](https://numpy.org/doc/stable/dev/internals.html#internal-organization-of-numpy-arrays)(aka the underlying C array) is set correctly. For example, calling `.values`
+  below produces a NumPy proxy array that
```

Comment:
I'm ok with changing the wording for now and updating it again after [more testing described here](https://github.com/rapidsai/cudf/issues/15397#issuecomment-2387464102).

Reply:
Yeah that's what I was thinking. We just talk about the conversion happening as the limitation. Once we address the testing you linked, we can add a note here saying that we work with the C API because we convert eagerly.